### PR TITLE
Bug 1570390 - UA override for zhidao.baidu.com, Bug 1570119 - Intervention for teamcoco.com and Bug 1570328 - Intervention for developer.apple.com

### PR DESCRIPTION
**Pull request by ksy36**
r? @denschub

### CODE REVIEW

File: src/injections/css/bug1570119-teamcoco.com-scrollbar-width.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+ * teamcoco.com - a scrollbar at the top covering navigation menu
+ * Bug #1570119 - https://bugzilla.mozilla.org/show_bug.cgi?id=1570119
+ *
+ * The scrollbar is covering navigation items making them unusable.
+ * There are ::-webkit-scrollbar css rules already applied to the scrollbar,
+ * hiding it in Chrome. Adding the scrollbar-width: none fixes the issue in Firefox.
+ */
+.css-bdnz85 {
+    scrollbar-width: none;
+}
```
Comment (ksy36) [inline src/injections/css/bug1570119-teamcoco.com-scrollbar-width.css]: Forgot to run prettier, it's fixed in the next commit.

File: src/injections/css/bug1570328-developer-apple.com-transform-scale.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * Bug #1570328 - https://bugzilla.mozilla.org/show_bug.cgi?id=1570328
+ * WebCompat issue #4070 - https://webcompat.com/issues/4070
+ *
+ * The site is relying on zoom property which is not supported by Mozilla,
+ * see https://bugzilla.mozilla.org/show_bug.cgi?id=390936. Adding a combination
+ * of transform: scale(1.4), transform-origin and width fixes the issue
+ */
+@media only screen and (min-device-width: 320px) and (max-device-width: 980px),
+  (min-device-width: 1024px) and (max-device-width: 1024px) and (min-device-height: 1366px) and (max-device-height: 1366px) and (min-width: 320px) and (max-width: 980px),
+  (min-device-width: 1024px) and (max-device-width: 1024px) and (min-device-height: 1366px) and(max-device-height: 1366px) and (min-width: 320px) and (max-width: 980px) {
```
Comment (denschub) [inline src/injections/css/bug1570328-developer-apple.com-transform-scale.css]: I have no idea why prettier does not care about this, but `and(max-device-height: 1366px)` is missing a space after the `and`.

Comment (denschub) [reply src/injections/css/bug1570328-developer-apple.com-transform-scale.css]: so we even need line 12? Isn't it exactly the same as the previous line? What am I missing? :)

Comment (ksy36) [reply src/injections/css/bug1570328-developer-apple.com-transform-scale.css]: Ah, good point. I've just copied it from the website's style sheet. I'll remove the last line

**Pull request merged**